contents/handbook/engineering/ee-setup.md

```diff
@@ -63,6 +63,23 @@ This setup is best if you want to iterate quickly on frontend changes. You get t
   - `export KAFKA_HOSTS=localhost:9092`
 - Run PostHog: `./bin/start`
 
+### Running ClickHouse, Kafka and Zookeeper on a Cloud Server
+
+This is useful if you have a Apple Silicon Mac.
+
+1. Get SSH access to any cloud server, e.g. a droplet from Digital Ocean.
```

Comment:
Just to make it super easy for anyone new, do want to write out how to get Digital Ocean access (maybe link to DO & say to create an account with posthog email if I recall correctly that was the way) & what droplet size they need to pick & that they should clean up the droplet later (we'd keep paying for it otherwise).

Reply:
Not sure about droplet size (I picked a middle-of-the road option), but operationally should people be putting these DO boxes on their company credit cards or is there a whole-team account?

Reply:
I didn't specify this as I didn't know. I just ran a demo on my own personal DO account to make sure the commands worked.

Reply:
For droplet size, again not sure. I just used the $20/mo option that comes with 4GB of RAM... yet I didn't do anything intense with it. A smaller one would probably have worked as well.

Reply:
@samwinslow We have a Posthog team in DigitalOcean, which I believe would be fine for everyone to use and probably better compared to using company cc (we can see you running something expensive and tell you or nuke it 😉 ). To join the team I don’t see a way to add anyone, maybe @fuziontech (as the owner) needs to invite you?